pkg/oci/load_balancer_security_lists.go

```diff
@@ -152,12 +152,20 @@ func (s *baseSecurityListManager) updateLoadBalancerRules(ctx context.Context, l
 
 		logger := s.logger.With("securityListID", *secList.Id)
 
-		lbEgressRules := getLoadBalancerEgressRules(logger, secList.EgressSecurityRules, nodeSubnets, ports.BackendPort, s.serviceLister)
-		lbEgressRules = getLoadBalancerEgressRules(logger, lbEgressRules, nodeSubnets, ports.HealthCheckerPort, s.serviceLister)
+		// -1 denotes nil ports.
```

Comment:
I need to thread though the 'current' backEnd and health ports in-order to carefully delete the leaking rules. As I mentioned at some point I would like to tidy up this double invocation to `getLoadBalancerEgressRules()` and make it work more like ` getNodeIngressRules()`.